aspnetcore/tutorials/choose-web-ui.md

```diff
@@ -3,83 +3,57 @@ title: Choose an ASP.NET Core UI
 author: wadepickett
 description: Learn when to use which ASP.NET Core web UI technologies. Understand the server, client and hybrid options.
 ms.author: wpickett
-ms.date: 09/12/2023
+ms.date: 10/04/2023
 uid: tutorials/choose-web-ui
 ---
 # Choose an ASP.NET Core web UI
 
 ASP.NET Core is a complete UI framework. Choose which functionalities to combine that fit the app's web UI needs.
 
-## Benefits vs. costs of server and client rendered UI
+## ASP.NET Core Blazor
 
-There are three general approaches to building modern web UI with ASP.NET Core:
+Blazor is a full-stack web UI framework and is recommended for most web UI scenarios.
 
-* Apps that render UI from the server.
-* Apps that render UI on the client in the browser.
-* Hybrid apps that take advantage of both server and client UI rendering approaches. For example, most of the web UI is rendered on the server, and client rendered components are added as needed.
+Blazor renders content with:
```

Comment:
I've reviewed this list about how Blazor renders content, but I'm not really sure that we need it here. I think what we really want is a list of Blazor benefits like we have in the other sections.

Reply:
Removed info on how Blazor renders.  Thanks.